src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -53,9 +53,10 @@ public function load(array $configs, ContainerBuilder $container)
 
         $config = $this->processConfiguration($configuration, $configs);
 
-        $container->setParameter('twig.exception_listener.controller', $config['exception_controller']);
+        $container->getDefinition('twig.exception_listener')->replaceArgument(0, $config['exception_controller']);
+        $container->getDefinition('twig.controller.preview_error')->replaceArgument(1, $config['exception_controller']);
 
-        $container->setParameter('twig.form.resources', $config['form_themes']);
```

Comment:
hmm, actually, I'm quite sure I know about bundles relying on this parameter to prepend form themes defining the rendering of their special blocks, so this is an actual BC break